docs/community/tutorials-contest/2022/avax-subnet-customization/README.md

```diff
@@ -0,0 +1,975 @@
+
+You can enable some precompiled contracts in the genesis file. These contracts provide God mode functionalities and are very useful for some cases. If you'd not like to use them, you can disable them by setting the config fields to null or by simply removing them from the genesis file.
+
+#### `contractDeployerAllowListConfig`
+
+This configuration allows you specify which addresses are authorized to deploy contracts. You can also customize this [in the last step of the Subnet wizard](#configure-contract-deployment-whitelist).
+If you'd like to restrict the contract deployer to a specific list of addresses, you can set the `contractDeployerAllowListConfig` to a JSON object with the following properties:
+
+```json
+"contractDeployerAllowListConfig": {
+  "blockTimestamp": 0,
+  "adminAddresses": ["0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC","0x0000...", "... and more"]
+}
+```
+
+Admin addresses can deploy new contracts and add new Admin and Deployer addresses. Precompiled contract is deployed to `0x0200000000000000000000000000000000000000`.
+[You can find more information from the Avalanche docs.](https://docs.avax.network/Subnets/customize-a-Subnet#restricting-smart-contract-deployers)
+
+
+#### `contractNativeMinterConfig`
+
+This configuration lets you use a smart contract to mint native tokens in your Subnet. This is useful if you want to use your own token minting system.  
+
+If you'd like to use a smart contract to mint native tokens, you can set the `contractNativeMinterConfig` to a JSON object with the following properties:
+
+```json
+"contractDeployerAllowListConfig": {
+  "blockTimestamp": 0,
+  "adminAddresses": ["0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC","0x0000...", "... and more"]
+}
+```
+
+Admin addresses can add new Admin and Minter addresses. Precompiled contract is deployed to `0x0200000000000000000000000000000000000001`. [You can find more information from the Avalanche docs.](https://docs.avax.network/Subnets/customize-a-Subnet#minting-native-coins)
+
+#### `txAllowListConfig`
+
+This configures allows you to specify which addresses are authorized to make transactions. It's especially useful if you're building a private network.
+If you'd like to restrict the transaction sender to a specific list of addresses, you can set the `txAllowListConfig` to a JSON object with the following properties:
+
+```json
+"txAllowListConfig": {
+  "blockTimestamp": 0,
+  "adminAddresses": ["0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC","0x0000...", "... and more"]
+}
+```
+
+Admin addresses can add new Admin and Allowed addresses. Precompiled contract is deployed to `0x0200000000000000000000000000000000000002`. [You can find more information from the Avalanche docs.](https://docs.avax.network/Subnets/customize-a-Subnet#restricting-who-can-submit-transactions)
+
+### Genesis Block
+This configuration is used to define the genesis block header. You generally don't need to change these. (except the `gasLimit` field) 
+
+#### `nonce`
```

Comment:
Per [Markdownlint](https://github.com/DavidAnson/markdownlint) Headings should be surrounded by blank lines.

> MD022/blanks-around-headings/blanks-around-headers: Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below]

[More info: MD022](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md022---headings-should-be-surrounded-by-blank-lines)